### ISSUE TITLE
Get Utils > Classes > ExtendsMultiple Working

### ISSUE THREAD

**Issue by andyslack** [juicyllama/framework#132]
We are trying to introduce a helper function which combined multiple Classes into one.

Code is here: https://github.com/juicyllama/framework/blob/main/common/utils/src/utils/Classes.ts

Use case: 
To be able to combine multiple Dtos, example: 
https://github.com/juicyllama/framework/blob/main/backend/websites/src/modules/websites/websites.dto.ts

**Please also introduce a test file for this usecase locally to ensure its working, combining multiple test Dtos.**

**Comment by andyslack** [comment]
No, I dont want to nest them, there should not be any duplicates, if so, we could print an error

**Comment by yossi-shasho** [comment]
@andyslack Alright. Is there any issue with the current implementation? whats exactly missing (apart from tests)?

**Comment by andyslack** [comment]
It's not working right now, it does not seem to return in the docs the correct full response

**Comment by yossi-shasho** [comment]
@andyslack I took the liberty change the signature of the ExtendsMultiple from `ExtendsMultiple([classA,...])` to `ExtendsMultiple(classA, ...)`

Also decided that the output class's constructor will take each base class's args as an array e.g. `new mixedClass([argA1, argA2...], [argB1,...])`. Hope that makes sense. I tried various other approaches but they all ended up clashing somehow.
